File: external/webroot/md/codeExample/Face-API-GroupAdduser/FaceAddUser.java
/*
 * Copyright (C) 2017 Baidu, Inc. All Rights Reserved.
 */
package com.baidu.ai.aip.face;

import com.baidu.ai.aip.utils.HttpUtil;

/**
 * 人脸查找——组内添加用户
 */
public class AddUser {

    /**
     * 重要提示代码中所需工具类
     * FileUtil,Base64Util,HttpUtil请从
     * http://ai.baidu.com/download/demo/java/utils/FileUtil.java
     * http://ai.baidu.com/download/demo/java/utils/Base64Util.java
     * http://ai.baidu.com/download/demo/java/utils/HttpUtil.java
     * 下载
     */
    public static void main(String[] args) {
        // 人脸查找——组内添加用户 url
        String addUserUrl = "https://aip.baidubce.com/rest/2.0/faceverify/v1/faceset/group/adduser";
        // 请求参数
        // 用户组
        String groupId = "test_group_2";
        // 用户ID
        String uid = "test_user_1";
        String params = "group_id=" + groupId + "&uid=" + uid;
        try {
            /**
             * 线上环境access_token有过期时间， 客户端可自行缓存，过期后重新获取。
             */
            String accessToken = "#####调用鉴权接口获取的token#####";
            String result = HttpUtil.post(addUserUrl, accessToken, params);
            System.out.println(result);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: external/webroot/md/codeExample/Face-API-GroupAdduser/FaceAddUser.cs
using System;
using System.IO;
using System.Net;
using System.Text;
using System.Web;

namespace com.baidu.ai
{
	public class AddUser
	{
		//组内添加用户
		public static String adduser()
		{
			string token = "########";
			string host = "https://aip.baidubce.com/rest/2.0/faceverify/v1/faceset/group/adduser?access_token=" + token;
			Encoding encoding = Encoding.Default;
			HttpWebRequest request = (HttpWebRequest)WebRequest.Create(host);
			request.Method = "post";
			request.ContentType = "application/x-www-form-urlencoded";
			request.KeepAlive = true;
			string str = "group_id=demo&uid=songqingyun2";
			byte[] buffer = encoding.GetBytes(str);
			request.ContentLength = buffer.Length;
			request.GetRequestStream().Write(buffer, 0, buffer.Length);
			HttpWebResponse response = (HttpWebResponse)request.GetResponse();
			StreamReader reader = new StreamReader(response.GetResponseStream(), Encoding.Default);
			string result = reader.ReadToEnd();
			Console.WriteLine("组内添加用户:");
			Console.WriteLine(result);
			return result;
		}
	}
}
